provider/source/data/debug/timezone/periods/v1.json

```diff
@@ -0,0 +1,4734 @@
+      "-4:00",
+      18
+    ]
+  },
+  "getbs": {
+    "1970-01-01 00:00": [
+      "+4:00/+5:00",
+      null
+    ],
+    "1991-03-30 22:00": [
+      "+3:00/+4:00",
+      null
+    ],
+    "1991-04-08 20:00": [
+      "+3:00/+4:00",
+      56
+    ],
+    "1994-09-24 20:00": [
+      "+4:00/+5:00",
+      56
+    ],
+    "2004-06-26 19:00": [
+      "+3:00/+4:00",
+      56
+    ],
+    "2005-03-26 23:00": [
+      "+4:00",
+      56
+    ]
+  },
+  "gfcay": {
+    "1970-01-01 00:00": [
+      "-3:00",
+      51
+    ]
+  },
+  "gggci": {
+    "1970-01-01 00:00": [
+      "+1:00",
+      null
+    ],
+    "1971-10-31 02:00": [
+      "+0:00/+1:00",
+      53
+    ]
+  },
+  "ghacc": {
+    "1970-01-01 00:00": [
+      "+0:00",
+      53
+    ]
+  },
+  "gigib": {
+    "1970-01-01 00:00": [
+      "+1:00/+2:00",
+      46
+    ]
+  },
+  "gldkshvn": {
+    "1970-01-01 00:00": [
+      "-3:00/-2:00",
+      60
+    ],
+    "1996-01-01 03:00": [
+      "+0:00",
+      53
+    ]
+  },
+  "glgoh": {
+    "1970-01-01 00:00": [
+      "-3:00/-2:00",
+      60
+    ],
+    "2023-03-26 01:00": [
+      "-2:00",
+      60
+    ],
+    "2024-03-26 01:00": [
+      "-2:00/-1:00",
+      58
+    ]
+  },
+  "globy": {
+    "1970-01-01 00:00": [
+      "-2:00/-1:00",
+      null
+    ],
+    "1981-03-29 02:00": [
+      "-1:00/+0:00",
+      59
+    ],
+    "2024-03-31 01:00": [
+      "-2:00/-1:00",
+      58
+    ]
+  },
+  "glthu": {
+    "1970-01-01 00:00": [
+      "-4:00/-3:00",
+      18
+    ]
+  },
+  "gmbjl": {
+    "1970-01-01 00:00": [
+      "+0:00",
+      53
+    ]
+  },
+  "gmt": {
+    "1970-01-01 00:00": [
+      "+0:00",
+      53
+    ]
+  },
+  "gncky": {
+    "1970-01-01 00:00": [
+      "+0:00",
+      53
+    ]
+  },
+  "gpbbr": {
+    "1970-01-01 00:00": [
+      "-4:00",
+      18
+    ]
+  },
+  "gpmsb": {
+    "1970-01-01 00:00": [
+      "-4:00",
+      18
+    ]
+  },
+  "gpsbh": {
+    "1970-01-01 00:00": [
+      "-4:00",
+      18
+    ]
+  },
+  "gqssg": {
+    "1970-01-01 00:00": [
+      "+1:00",
+      6
+    ]
+  },
+  "grath": {
+    "1970-01-01 00:00": [
+      "+2:00/+3:00",
+      47
+    ]
+  },
+  "gsgrv": {
+    "1970-01-01 00:00": [
+      "-2:00",
+      126
+    ]
+  },
+  "gtgua": {
+    "1970-01-01 00:00": [
+      "-6:00/-5:00",
+      9
+    ]
+  },
+  "gugum": {
+    "1970-01-01 00:00": [
+      "+10:00/+11:00",
+      null
+    ],
+    "2000-12-22 14:00": [
+      "+10:00",
+      32
+    ]
+  },
+  "gwoxb": {
+    "1970-01-01 00:00": [
+      "-1:00",
+      null
+    ],
+    "1975-01-01 01:00": [
+      "+0:00",
+      53
+    ]
+  },
+  "gygeo": {
+    "1970-01-01 00:00": [
+      "-3:45",
+      62
+    ],
+    "1975-08-01 03:45": [
+      "-3:00",
+      62
+    ],
+    "1992-03-29 04:00": [
+      "-4:00",
+      62
+    ]
+  },
+  "hebron": {
+    "1970-01-01 00:00": [
+      "+2:00/+3:00",
+      74
+    ],
+    "1995-12-31 22:00": [
+      "+2:00/+3:00",
+      47
+    ]
```

Comment:
Observation: A time zone that has always had the same offsets, but only adopted a metazone name in 1995.

Also, why is this 12-31 at 22:00? Seems like it should probably be 01-01 at 00:00